CoroutineTemplate/build.gradle

```diff
@@ -0,0 +1,77 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        build_gradle_version = '4.1.1'
+
+        android_version_code = 9
+        android_version_name = "0.7.0"
+        android_min_sdk_version = 21
+        android_target_sdk_version = 28
+        android_compile_sdk_version = 28
+
+        android_support_version = '1.1.0'
+        constraint_layout_version = '2.0.0-alpha3'
+        androidx_core_ktx_version = '1.3.0'
+
+        kotlin_version = '1.4.30'
+        kotlinx_coroutines_version = '1.4.1'
+
+        android_test_junit_version = '1.1.1'
+        test_espresso_version = "3.1.0"
+        test_junit_version = '4.13'
```

Comment:
please keep them in a correct order, test components should start with test prefix